dictionary/README.md

```diff
@@ -2,4 +2,4 @@
 Please write a couple of sentences explaining your choice of restart
 strategy.
 
-
+The reasoning behind my choice of the restart strategy being ":transient" is due to the fact that if the Dictionary's dependencies want to close normally, the Dictionary should do the same. However, in the case of an irregular shutdown, the Dictionary should be restarted to be ready to be used again.
```

Comment:
# -2

I'd prefer :permanent here. The dictionary will never normally terminate unless the entire Beam node stops.